### DISCUSSION TITLE
Silhouette Updates

### DISCUSSION THREAD

**Discussion by aa0pb** [vradarserver/vrs#51] [Q&A]
Howdy,
Where do you get the updated silhouettes?  E75L is one example of missing.

Thanks,
Phil

**Comment by vradarserver** [comment]
I didn't draw any of the silhouettes that are knocking around. The bulk of the ones that I've seen were drawn by Ian Kirby, his site is here but it looks like it hasn't been updated in a while:

http://www.iksilhouettes.co.uk/

**Comment by rikgale** [comment]
Dragging up a bit of an old thread, but as I stumbled across it I thought I should add some useful information.

I maintain and actively update my own silhouettes for VRS, all redone from scratch with attention paid to scaling and consistency. This is a very comprehensive collection with nearly 1800 silhouettes. Regular updates are posted on my github repo https://github.com/rikgale/VRSOperatorFlags and on the radarspotting.com forum gallery.

Hopefully you will find this useful.

**Reply by vradarserver** [reply]
That's actually pretty cool - and there's an E75L in there as well to boot :) Thanks for that.
